Openshift-EE/pipelines/Cassandra/stages/4-chaos/M0NH-jiva-app-kill/jiva-app-kill.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+set -x
+
+pod() {
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'cd e2e-openshift && bash Openshift-EE/pipelines/Cassandra/stages/4-chaos/M0NH-jiva-app-kill/jiva-app-kill node '"'$CI_JOB_ID'"'' '"'$CI_PIPELINE_ID'"' '"'$CI_COMMIT_SHA'"'
+}
+
+node() {
+
+#Gitlab job id obtain from the gitlab env ($CI_JOB_ID).
+job_id=$1
+
+# Gitlab pipeline id obtained from gitlab env ($CI_PIPELINE_ID).
+pipeline_id=$2 
+
+#Gitlab commit id Obtained fron gilab env ($CI_COMMIT_SHA).
+commit_id=$3 
+
+#test case ID.
+case_id=M0NH
+
+#github token to push the test result into github mayadata-io/e2e-openshift repository.
+#This token is set as an env in ~/.profile file in the test cluster.
+source ~/.profile
+gitToken=$github_token
+
+time="date"
+current_time=$(eval $time)
+
+present_dir=$(pwd)
+echo $present_dir
+
+bsah Openshift-EE/utils/pooling jobname:cstor-volume-mgmt-kill
+bash Openshift-EE/utils/e2e-cr jobname:jiva-app-kill jobphase:Running init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+
+################
+# LitmusBook 1 #
+################
+
+#### Generating test name using test case name
+
+run_id="jiva";test_name=$(bash Openshift-EE/utils/generate_test_name testcase=application-pod-failure metadata="")
```

Comment:
@w3aman can you include run_id in metadata.

Reply:
Done !